tests/__init__.py

```diff
@@ -46,7 +46,7 @@ def _test_under_locale(testself, sftp):
                 try_locales.pop(0)
             else:
                 try:
-                    return test_method(testself)
+                    return test_method(testself, sftp)
```

Comment:
I'm relatively unfamiliar with the guts of the test suite, but what does adding the `sftp` here accomplish? I don't see anything obvious in any of the other changes where this is used.